### ISSUE TITLE
[regression v3]: incorect validation of select

### ISSUE THREAD

**Issue by karlismelderis-mckinsey** [oguimbal/pg-mem#412]
# Describe the bug

I didn't check number conversion to string but we have tests where loo long string was passed down to TypeORM and tests passed for v2 pg-mem

Example:
Table has varchar column with max length set to 1

### Postgres:
returns empty result

## PG-mem:
```
Error: value too long for type character varying(36)
    
    🐜 This seems to be an execution error, which means that your request syntax seems okay,
        but the resulting statement cannot be executed → Probably not a pg-mem error.
    
    *️⃣ Failed SQL statement: SELECT ...
```

### Postgres:
ERROR:  operator does not exist: character varying = integer

## PG-mem:
No error

### Postgres:
ERROR:  operator does not exist: character varying = integer

### PG-mem:
Error: value too long for type character varying(1)
// seems like number got converted to string 🤔

# pg-mem version
3.0.2

**Comment by oguimbal** [comment]
Thanks ! Should be fixed in `pg-mem@3.0.3`

**Comment by karlismelderis-mckinsey** [comment]
this is fixed
💪
thank you!
